src/main/java/org/openlmis/notification/service/UserContactDetailsService.java

```diff
@@ -48,6 +71,57 @@ public UserContactDetails addOrUpdate(UserContactDetails details) {
         : addUserContactDetails(details);
   }
 
+  /**
+   * Saves user contact details.
+   *
+   * @param userDto user contact details object
+   * @param successfulResults list with success results
+   * @param failedResults list with failed results
+   */
+  public void saveUsersContactDetails(UserContactDetailsDto userDto,
```

Comment:
Please refactor this so it doesn't return result through parameters, something like:

```
public Result saveUsersContactDetails(UserContactDetailsDto userDto) {}
...
class Result {
   String error;
   boolean isSuccess() { return error == null; }
}
```